data/containers/autoyast_containers.xml.ep

```diff
@@ -29,8 +19,22 @@
         <arch>{{ARCH}}</arch>
       </addon>
       % }
+      <addon>
+        <name>sle-module-containers</name>
+        % if ($get_var->('VERSION') =~ /12-SP5|12-SP4|12-SP3/) {
+        <version>12</version>
```

Comment:
Before you ask, for all the 12-SPx versions, we need to hardcode `12` here. Using the variable fails, e.g. `<version>12.5</version>` will through an error (see scc.suse.com for more info).